LAUNCH.md

```diff
@@ -0,0 +1,200 @@
+  --help-md             Show an extended help message as markdown. Can be
+                        useful to overwrite LAUNCH.md with `$ python launch.py
+                        --help-md > LAUNCH.md`
+  --job_name JOB_NAME   slurm job name to show in squeue. Defaults to crystal-
+                        gfn
+  --outdir OUTDIR       where to write the slurm .out file. Defaults to
+                        $SCRATCH/crystals/logs/slurm
+  --cpus_per_task CPUS_PER_TASK
+                        number of cpus per SLURM task. Defaults to 2
+  --mem MEM             memory per node (e.g. 32G). Defaults to 32G
+  --gres GRES           gres per node (e.g. gpu:1). Defaults to gpu:1
+  --partition PARTITION
+                        slurm partition to use for the job. Defaults to long
+  --modules MODULES     string after 'module load'. Defaults to anaconda/3
+                        cuda/11.3
+  --conda_env CONDA_ENV
+                        conda environment name. Defaults to gflownet
+  --venv VENV           path to venv (without bin/activate). Defaults to None
+  --code_dir CODE_DIR   cd before running main.py (defaults to here). Defaults
+                        to ~/ocp-project/gflownet
+  --jobs JOBS           run file name in external/jobs (without .yaml).
+                        Defaults to None
+  --dev                 Don't run just, show what it would have run. Defaults
+                        to False
+  --verbose             print templated sbatch after running it. Defaults to
+                        False
+  --force               skip user confirmation. Defaults to False
+
+```
+
+## 🎛️ Default values
+
+```yaml
+code_dir      : ~/ocp-project/gflownet
+conda_env     : gflownet
+cpus_per_task : 2
+dev           : False
+force         : False
+gres          : gpu:1
+job_name      : crystal-gfn
+jobs          : None
+main_args     : None
+mem           : 32G
+modules       : anaconda/3 cuda/11.3
+outdir        : $SCRATCH/crystals/logs/slurm
+partition     : long
+template      : /Users/victor/Documents/Github/gflownet-dev/sbatch/template-conda.sh
+venv          : None
+verbose       : False
+```
+
+## 🥳 User guide
+
+In a word, use `launch.py` to fill in an sbatch template and submit either
+a single job from the command-line, or a list of jobs from a `yaml` file.
+
+Examples:
+
+```sh
+# using default job configuration, with script args from the command-line:
+$ python launch.py user=$USER logger.do.online=False
+
+# overriding the default job configuration and adding script args:
+$ python launch.py --template=sbatch/template-venv.sh \
+    --venv='~/.venvs/gfn' \
+    --modules='python/3.7 cuda/11.3' \
+    user=$USER logger.do.online=False
+
+# using a yaml file to specify multiple jobs to run:
+$ python launch.py --jobs=jobs/comp-sg-lp/v0" --mem=32G
+```
+
+### 🤓 How it works
+
+1. All experiment files should be in `external/jobs`
+    1. Note that all the content in `external/` is **ignored by git**
```

Comment:
Do you wanna add this change in `.gitignore` to this PR?

Reply:
I thought it already was but maybe it stayed in Dave. I'll fo that indeed